.cirrus.yml

```diff
@@ -118,11 +118,11 @@ macos_task:
               RUST: stable
         # This job tests our minimum supported Rust version, so only bump on
         # Newsboat release day
-        - name: macOS GCC Rust 1.40.0
+        - name: macOS GCC Rust 1.42.0
```

Comment:
I got a bit confused by the commit message stating that this bumps the MSRV to 1.40.0 (instead of 1.42.0 which is shown here)

Reply:
Oh, looks like I wasn't paying enough attention. Thanks! I'll have to force-push to amend that.